### PR TITLE
Cache Correction

### PR DESCRIPTION
**Pull request by EmilyNecciai**
# Pull Request/Issue Resolution

**Description of Change:**

**Reason for Change:**

Closes #**ISSUE_NUMBER_HERE**

---
---

## PR Checklist
- [ ] Ensure you have completed our CLA.
- [ ] Tag @EmilyNecciai as a reviewer when the your work is done and ready to be reviewed for merge. 
- [ ] Consult the [Docs Text Formatting Guide](https://github.com/Appboy/success/wiki/Docs-Text-Formatting-Guide) to be sure you're utilizing the proper markdown formatting.
- [ ] Consult the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/success/wiki/Writing-Style-Guide-&-Best-Practices) to be sure you're aligning with our voice and other style best practices.
- [ ] [Preview your deployed changes](https://homeslice.braze.com/docs) to confirm that none of your changes break production. Pay close attention to links and images.
- [ ] Tag others as Reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

---
---

### CODE REVIEW

**Comment by EmilyNecciai** [comment]
@mkigikm I got a request to update this info. I'm not sure that I'm depicting it correctly and I believe I need an example for the newer portion (re: cache busting). Could you take a look and let me know if you have any suggestions or improvements?

File: _docs/_user_guide/personalization_and_dynamic_content/connected_content/local_connected_content_variables.md [hunk 1]
```diff
@@ -94,18 +94,24 @@ You can utilize the HTTP status from a Connected Content call by first saving it
...
 
 ```
 #  Will cache for 900 seconds (15 minutes)
-{% connected_content https://example.com/webservice.json :cache 900 %}
+{% connected_content https://example.com/webservice.json :cache_max_value 900 %}
 ```
 
 {% endraw %}
 
+If you don't want to cache a value at all, to create a cache bust, use a POST endpoint with a timeframe stamp or `braze_id`.
```
Comment (mkigikm) [inline _docs/_user_guide/personalization_and_dynamic_content/connected_content/local_connected_content_variables.md]: With a POST you don't need to cache bust at all. We never cache the results from POST requests.

File: _docs/_user_guide/personalization_and_dynamic_content/connected_content/local_connected_content_variables.md [hunk 2]
```diff
@@ -87,26 +93,45 @@ You can optionally provide a POST body by specifying `:body` followed by a query
 
 You can utilize the HTTP status from a Connected Content call by first saving it as a local variable and then using the `__http_status_code__` key. For example:
 
-```html
+```js
 {% connected_content https://example.com/api/endpoint :save result %}
```
Comment (zzhaobraze) [inline _docs/_user_guide/personalization_and_dynamic_content/connected_content/local_connected_content_variables.md]: @EmilyNecciai this needs to be wrapped in a raw tagged also

Comment (EmilyNecciai) [reply _docs/_user_guide/personalization_and_dynamic_content/connected_content/local_connected_content_variables.md]: Oops! Thank you!

**Pull request merged**